src/GenAPI/Microsoft.DotNet.GenAPI/SyntaxRewriter/FieldDeclarationSyntaxRewriter.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.DotNet.GenAPI.SyntaxRewriter
+{
+    /// <summary>
+    /// Removes static keyword if field has both static and const modifiers.
+    /// Fixes the https://github.com/dotnet/arcade/issues/11934 issue.
+    /// </summary>
+    public class FieldDeclarationSyntaxRewriter : CSharpSyntaxRewriter
```

Comment:
Curious, why do we need these types? Can't we add that functionality into the CSharpSyntaxWriter? Asking as I'm slightly concerned about allocation. How often will these types be instantiated?

Reply:
> It's allocated once per assembly.
one dynamic library - one allocation.